src/catalog/schema_utils.py

```diff
@@ -78,3 +82,24 @@ def get_petastorm_schema(name, column_list):
 
         petastorm_schema = Unischema(name, petastorm_column_list)
         return petastorm_schema
+
+    @staticmethod
+    def petastorm_type_cast(schema: Unischema, df: pd.DataFrame) \
```

Comment:
Why do we need this?

Reply:
The reason is the same input array say [1,2,3,4,5] can be interpreted as different types (UINT8, INT8, INT16 and etc.), so we need to cast the array to the user defined type.

Reply:
Got it